crates/loader/src/loader.rs

```diff
@@ -323,6 +324,8 @@ pub struct LanguageConfiguration<'a> {
     highlight_names: &'a Mutex<Vec<String>>,
     #[cfg(feature = "tree-sitter-highlight")]
     use_all_highlight_names: bool,
+    #[cfg(not(any(feature = "tree-sitter-highlight", feature = "tree-sitter-tags")))]
+    _phantom: PhantomData<&'a ()>,
```

Comment:
Does this need to be conditional on the features? Can we just always have the phantom data?

Reply:
It's not needed if the highlight feature is enabled, which uses the lifetime.

Reply:
Right, but the `PhantomData` has zero cost; what we want to avoid is unnecessary cost. So we should only add the conditional compilation if it *needs* to be there. Does the code compile ok with both the highlight feature *and* the `PhantomData`? If so, we should just always include the phantom data.